test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -869,7 +865,7 @@ def test_successful_fast_fund_credit_with_token_on_cft_gateway
 
   def test_failed_fast_fund_credit_on_cft_gateway
     options = @options.merge({ fast_fund_credit: true })
-    refused_card = credit_card('4917300800000000', name: 'REFUSED') # 'magic' value for testing failures, provided by Worldpay
+    refused_card = credit_card('4444333322221111', name: 'REFUSED') # 'magic' value for testing failures, provided by Worldpay
```

Comment:
This updated test card number is the one listed in the worldpay docs for fast_fund testing. Test was failing with the previous number.